decidim-core/app/views/decidim/widgets/show.js.erb

```diff
@@ -0,0 +1,3 @@
+document.write('<link type="text/css" rel="stylesheet" media="all" href="<%= asset_url 'decidim/widget.css' %>"></link>');
+
+document.write('<div class="decidim-widget"><%= j(render 'widget') %><div class="organization"><%= j(render partial: "layouts/decidim/logo", locals: { organization: current_organization }) %></div></div>');
```

Comment:
If we end up using iframes, I'd create a `widget` layout with the needed base CSS and JS and just use `src="<%= url_for(format: 'html')` in here.